libafl/src/executors/hooks/timer.rs

```diff
@@ -242,6 +242,11 @@ impl TimerStruct {
     #[cfg(target_os = "linux")]
     #[must_use]
     /// Constructor but use batch mode
+    /// Timeout mechanism with imprecise timing.
```

Comment:
Maybe also note that performance should be better? Else why would anyone use it :D

Reply:
Claude sonnet missed that piece of information